src/runtime/gc_boehm.c

```diff
@@ -0,0 +1,16 @@
+//go:build none
```

Comment:
Why is the `none` here if this is included for "included in the build on systems that support the Boehm GC."?

Reply:
It's the same for other files such as src/runtime/signal.c, src/runtime/os_darwin.c, etc. Instead of using the normal way (which would lead to CGo errors), it is manually added in compileopts/target.go.

```diff
		spec.ExtraFiles = append(spec.ExtraFiles,
			"src/internal/futex/futex_linux.c",
+			"src/runtime/gc_boehm.c",
			"src/runtime/runtime_unix.c",
			"src/runtime/signal.c")
```

Reply:
Update: actually I just removed this file and did the same thing in a different/simpler way.